Serializer/Normalizer/IterableNormalizer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * This file is part of the GDPR bundle.
+ *
+ * @category  Bundle
+ * @package   Gdpr
+ * @author    SuperBrave <info@superbrave.nl>
+ * @copyright 2018 SuperBrave <info@superbrave.nl>
+ * @license   https://github.com/superbrave/gdpr-bundle/blob/master/LICENSE MIT
+ * @link      https://www.superbrave.nl/
+ */
+
+namespace Superbrave\GdprBundle\Serializer\Normalizer;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerAwareTrait;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalizes data if it's iterable by calling the normalizer chain.
+ *
+ * @author Jelle van Oosterbosch <jvo@superbrave.nl>
+ */
+class IterableNormalizer implements NormalizerInterface, NormalizerAwareInterface
+{
+    use NormalizerAwareTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        if (true === is_object($data) &&
+            (is_array($data) || (is_object($data)) && ($data instanceof \Traversable))
```

Comment:
This entire check is iffy at best; you check whether $data is an object twice (once stand-alone and once again in a wrapped portion), the wrapped portion also checks for "is $data an array or object and traversable?" which is hard to read and vague in its resulolution order.

Please rewrite/reorder the checks, or maybe even nest them for clarity.

Reply:
Agreed with above

Reply:
This is a polyfill for the `is_iterable` function - will be replaced when I merge this change to version 2.0.